names/slugs_test.go

```diff
@@ -56,6 +56,13 @@ func TestImageInfoToFriendlyName(t *testing.T) {
 			"quay.io-matthiasb-1-kubevuln-renaming-54fbec",
 			nil,
 		},
+		{
+			"Image ID format with at sign works",
+			"docker.io/kindest/local-path-provisioner:v0.0.23-kind.0@sha256:f2d0a02831ff3a03cf51343226670d5060623b43a4cfc4808bd0875b2c4b9501",
+			"docker.io/kindest/local-path-provisioner:v0.0.23-kind.0@sha256:f2d0a02831ff3a03cf51343226670d5060623b43a4cfc4808bd0875b2c4b9501",
+			"docker.io-kindest-local-path-provisioner-v0.0.23-kind.0-sha256-f2d0a02831ff3a03cf51343226670d5060623b43a4cfc4808bd0875b2c4b9501-4b9501",
```

Comment:
this is so ugly...

Reply:
Ah, yes, a valid and human-readable name which provides a fantastic user experience!